low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -880,9 +900,13 @@ def _generate_disk_space_alert(self):
         if self._node_sensor.disk_used_percentage <= self._disk_usage_threshold \
            and self.high_usage['disk']:
 
-            # Create the disk space data message and hand it over to the egress processor to transmit
-            fault_resolved_event = "Disk usage decreased to %s, lesser than configured threshold of %s" \
-                                %(self._node_sensor.disk_used_percentage, self._disk_usage_threshold)
+            # Create the disk space data message and hand it over
+            # to the egress processor to transmit
+            fault_resolved_event = "Disk usage has decreased to {}%, "\

```

Comment:
lets keep minimal grammar, avoid full stop even. Use 'lower' instead of lesser
> "Disk usage decreased to {}%, lower  than the configured threshold of {}%"

same comment applies everywhere else

Reply:
Changed lesser to lower, keeping the punctuation as discussed.